_content/essays/concise-spiritual-advice_sera-khandro.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Concise Spiritual Advice"
+authors:
+  - "Sera Khandro"
+subcat: poetry
+translator: "Christina Monson"
+external_url: "https://www.lotsawahouse.org/tibetan-masters/sera-khandro/concise-spiritual-advice"
+drive_links:
+  - "https://drive.google.com/file/d/1k6B2oOGA2ywnj8PoE6ja073woYCXh6Kv/view?usp=sharing"
+course: cosmology
+tags:
+  - anicca
+  - karma
+  - sati
+year: 2020
+publisher: Lotsawa House
+pages: 2
+---
+
+> Pray to your master and to the Three Jewels,
+and strive to be wholesome – physically, verbally and mentally.
+ to remain in solitude, establish mindfulness, and develop bodhicitta.
+In this brief poem, the great Tulku and Yogini, Sera Khandro, exhorts readers to wholeheartedly practice the Dharma. Khandro points out the importance of impermanence and karma to help practitioners overcome attachments and develop wholesome behavior. Other pieces of advice are
```

Comment:
Your summary ends in a sentence fragment?

Two spaces at the ends of the lines of poetry above are needed to tell Markdown those linebreaks are important.

And there must be at least one blank line between the quote and the summary 😊